git/util.py

```diff
@@ -591,7 +591,7 @@ def default_email():
             return user_id
 
         def default_name():
-            default_email().split('@')[0]
+            return default_email().split('@')[0]
```

Comment:
It’s funny this slipped by me as well since I only write Rust these says, in which this is exactly what we would have expected.
Maybe I have just been a sloppy reviewer though 😅

Reply:
Thanks for catching this one early, @harupy!